modules/config-updates-39.adoc

```diff
@@ -114,4 +114,30 @@ FEATURE_GARBAGE_COLLECTION: true
 PERMANENTLY_DELETE_TAGS: true
 QUOTA_TOTAL_DELAY_SECONDS: 1800
 RESET_CHILD_MANIFEST_EXPIRATION: true
-----
\ No newline at end of file
+----
+
+[id=postgresql-pvc-backup-config-fields]
+== PostgreSQL PVC backup configuration fields
+
+The following configuration field has been added to configure whether {productname} automatically removes old persistent volume claims (PVCs) when upgrading from version 3.8 -> 3.9:
```

Comment:
This is not a configuration field. This is an environment variable for the quay operator. It can be set via the `Subscription` object.

Reply:
not sure everybody know how, I post a Subscription example:
```
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: quay-operator
  namespace: quay-enterprise
spec:
  channel: stable-3.8
  name: quay-operator
  source: redhat-operators
  sourceNamespace: openshift-marketplace
  config:
    env: 
    - name: POSTGRES_UPGRADE_RETAIN_BACKUP
      value: "true"
```
pls correct me if I am wrong @dmage